docker/Dockerfile

```diff
@@ -15,5 +15,25 @@ RUN mkdir plugins/oss-fs-hadoop && \
     ln -s ../../opt/flink-oss-fs-hadoop-1.15.2.jar plugins/oss-fs-hadoop/flink-oss-fs-hadoop-1.15.2.jar && \
     ln -s ../../opt/flink-s3-fs-hadoop-1.15.2.jar plugins/s3-fs-hadoop/flink-s3-fs-hadoop-1.15.2.jar
 
+# install python3: it has updated Python to 3.9 in Debian 11 and so install Python 3.7 from source
```

Comment:
Install Python 3.7. This is because flink:1.15.2 docker image has been upgraded to use Debian 11 which uses Python 3.9. But PyFlink only supports Python 3.6, 3.7 and 3.8.